lib/jekyll-github-metadata/ghp_metadata_generator.rb

```diff
@@ -34,19 +34,39 @@ def git_remote_url
         end.uniq.first || ""
       end
 
-      def git_nwo
+      def nwo_from_git_origin_remote
         return unless Jekyll.env == "development"
         matches = git_remote_url.match %r{github.com(:|/)([\w-]+)/([\w-]+)}
```

Comment:
@benbalter What if we use [`Pages. github_hostname`](https://github.com/jekyll/github-metadata/blob/5e38433ca027efbb3491c6e04e8a844cdc5b56e6/lib/jekyll-github-metadata/pages.rb#L59-L61)